dictionaries/condensed-strokes.json

```diff
@@ -237,6 +240,7 @@
 "EPB/TKPWROS/-G": "engrossing",
 "EPB/TRAOET/KWREUS": "entreaties",
 "EPL/PWAES/-S": "embassies",
+"EPL/S-PTKARB": "em dash",
```

Comment:
`S-PTKARB` doesn't appear to be in steno order. Are you missing a `/` @paulfioravanti ?

Reply:
Thanks for that, @didoesdigital. Definitely a typo: should be `EPL/S-P/TKARB`. Will fix.